### DISCUSSION TITLE
error 404 before the homepage

### DISCUSSION THREAD

**Discussion by leo-jet** [quasarframework/quasar#10432] [CLI - SSR mode]
Hi, 
I am building a ssr website, but when I deploy it, the 404 page show first before the home. I precise I have deployed the started code whithout modification.
I use quasar v1.
this is an example : https://prepabac.homeschool-cameroon.com/

**Comment by ontwikkelfabriek** [comment]
@leo-jet 

If you locally deploy the app using `quasar serve` on the dist folder ( the build folder). What happens then?
